### DISCUSSION TITLE
What is Proof of Stack

### DISCUSSION THREAD

**Discussion by nehal2210** [nehal2210/Blockchain-Engineer-Course#2] [Q&A] [answered]
Now  Ethereum works on proof of stack, so we need to discuss it.
* it's working mechanism
* its advantages
* its disadvantages
* How can we take advantage of it

**Comment by nasrullahd** [comment]
Now Ethereum convert Prof of stack so it's advantage

**Reply by nehal2210** [reply]
please research about it, and explain it in detail.

**Reply by Maze004** [reply]
> Now Ethereum convert Prof of stack so it's advantage

what is the advantage if Ethereum converted to PoS

**Comment by imtiaz70** [comment]
POS save energy

**Reply by nehal2210** [reply]
please research about it, and explain in detail.

**Reply by ahmdhsnkhan** [reply, marked as answer]
> POS save energy

- 1. PoS protocols are much more [energy efficient] (https://blocklr.com/bitcoin/bitcoin-mining-7x-less-energy-aluminum-mining/) since they eliminate the need for computational-intensive mining.
